src/caret_analyze/architecture/architecture.py

```diff
@@ -390,6 +431,19 @@ def insert_variable_passing(self, node_name: str,
 
     def remove_publisher_callback(self, node_name: str,
                                   publish_topic_name: str, callback_name: str) -> None:
+        """
+        Remove callback to publisher in "node_name" node.
```

Comment:
Please clarify whether this function to remove the publisher's callback or both the publisher and its callback.

Reply:
This function deletes "which CB is registered with the publisher".